pandas/tests/sparse/test_series.py

```diff
@@ -58,6 +60,9 @@ def _test_data2_zero():
 
 class TestSparseSeries(SharedWithSparse):
 
+    series_klass = SparseSeries
+    _assert_series_equal = staticmethod(tm.assert_sp_series_equal)
```

Comment:
hmm, I like that actually.

The only thing is we are using ``self._assert_series_equal``, which is a tiny bit annoying, when in the rest of the codebase we are using ``tm.assert_series_equal`` (or ``tm.assert_sp_series_equal``.

any ideas on a better / more consistent way would be good. (maybe add a comment to this effect in the sub-classes about this)